### ISSUE TITLE
Remove batch norm from trx ecoder

### ISSUE THREAD

**Issue by ivkireev86** [dllllb/pytorch-lifestream#128]
Leak for events predictive tasks.
